extensions/fathom-analytics-stats/src/open-site-settings.tsx

```diff
@@ -0,0 +1,10 @@
+import { getPreferenceValues, open } from "@raycast/api";
+
+interface Preferences {
+  apiToken: string;
+  siteId: string;
+}
```

Comment:
```suggestion
```
You can remove this in all files since it's already automatic generated by Raycast's `raycast-env.d.ts`